packages/backend/server/src/schema.gql

```diff
@@ -554,8 +559,11 @@ type Query {
 
 input QueryChatHistoriesInput {
   action: Boolean
+  fork: Boolean
   limit: Int
+  messageOrder: [ChatHistoryOrder!]
```

Comment:
forget to regenerate type? why array here

Reply:
fixed

Reply:
client as well

Reply:
done